app/src/main/java/com/nervousfish/nervousfish/activities/MainActivity.java

```diff
@@ -41,6 +45,7 @@
     protected void onCreate(final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        LOGGER.info("MainActivity entered");
```

Comment:
Maybe change this to `MainActivity created` and move it to the end of the method for clarity